server/app/services/export/CsvExporter.java

```diff
@@ -70,13 +74,14 @@ public void exportRecord(
           printer.print(application.getApplicantData().preferredLocale().toLanguageTag());
           break;
         case CREATE_TIME:
-          printer.print(application.getCreateTime().toString());
+          printer.print(dateConverter.renderDateTimeDataOnly(application.getCreateTime()));
           break;
         case SUBMIT_TIME:
-          printer.print(
-              application.getSubmitTime() != null
-                  ? application.getSubmitTime().toString()
-                  : EMPTY_VALUE);
+          if (application.getSubmitTime() == null) {
+            printer.print(EMPTY_VALUE);
+            break;
+          }
+          printer.print(dateConverter.renderDateTimeDataOnly(application.getSubmitTime()));
```

Comment:
Can you move this into a private method of it's own? It's a little hard to read having multiple breaks in the same case.

Reply:
Good point. Actually, I've changed it to an if...else, which looks more readable to me and keeps the logic in place.
Let me know in case it still looks a bit cryptic.